chromium/background-scripts/rules.js

```diff
@@ -722,6 +722,36 @@ RuleSets.prototype = {
     return false;
   },
 
+  /**
+   * Get a list of simple rules (active, with no exclusions) for all hosts that
+   * are in a single ruleset, and end in the specified ending.
+   * @param ending Target ending to search for
+   * @returns A list of { host, from_regex, to, scope_regex }
+   */
+  getSimpleRulesEndingWith: function(ending) {
+    let results;
+
+    if (this.wasm_rs) {
+      results = this.wasm_rs.get_simple_rules_ending_with(ending);
+    } else {
+      results = [];
+      for(let [host, rulesets] of this.targets) {
+        if (host.endsWith(ending) &&
+            rulesets.length == 1 &&
+            rulesets[0].active == true &&
```

Comment:
```suggestion
            rulesets[0].active &&
```

Reply:
`active` should either be true or false in rulesets, and I'd like to maintain parity with the [`lib-core` implementation](https://github.com/EFForg/https-everywhere-lib-core/blob/master/src/rulesets.rs#L387) here, but this should probably be `===` instead.